docs/content/en/docs/getting-started/install-k8s.md

```diff
@@ -96,18 +96,16 @@ parameters.
 
 ### Deploy demo application
 
-To explore Tetragon its helpful to have a sample workload. Here we use the Cilium
-HTTP application, but any workload would work equally well.
-
-To use our [demo
-application](https://docs.cilium.io/en/v1.11/gettingstarted/http/#deploy-the-demo-application)
+To explore Tetragon it is helpful to have a sample workload. Here we use Cilium's
+[demo application](https://docs.cilium.io/en/v1.14/gettingstarted/demo/),
+but any workload would work equally well:
 
 ```shell
-kubectl create -f {{< demo-app-url >}}
+kubectl create -f https://raw.githubusercontent.com/cilium/cilium/1.14.4/examples/minikube/http-sw-app.yaml
```

Comment:
So this URL is located in the `hugo.toml` file of the doc because I think it's used elsewhere, you can update it there.

Reply:
Thanks for the pointer, I was not aware of the `hugo.toml` file. Will update it there.

Reply:
Actually, the current value of `demo_app_url` points to version `v1.15.0-pre.1` which is more recent compared to `1.14.4`.

However, it seems that this URL is not correctly rendered, regardless its value in `hugo.toml`:

https://tetragon.io/docs/getting-started/install-k8s/#deploy-demo-application

Reply:
IIUC and since we are using [Hugo's shortcodes](https://gohugo.io/content-management/shortcodes/#param) we have to use parameters in `.md` files.

Maybe this commit is related? cca76556fb55810ee9a239d83e4f165d9c4fa5ae

Moving `demo_app_url` under the `[params]` section does the trick.

Reply:
We should probably move this discussion to https://github.com/cilium/tetragon/pull/1965.